contracts/GnosisSafe.sol

```diff
@@ -57,7 +59,7 @@ contract GnosisSafe
     mapping(address => mapping(bytes32 => uint256)) public approvedHashes;
 
     // This constructor ensures that this contract can only be used as a master copy for Proxy contracts
-    constructor() public {
+    constructor() {
```

Comment:
Constructor is public by default, right?

Reply:
yes, that was a change with 0.7 I think (or even 0.6)